packages/main/test/specs/DatePicker.spec.js

```diff
@@ -1192,4 +1192,86 @@ describe("Date Picker Tests", () => {
 
 		assert.equal(await input.getProperty("valueState"), "Error", "value state of the input is valid (3)");
 	});
+
+	it("should open calendar picker in CalendarMode.DAY_MONTH_YEAR mode", async () => {
+		datepicker.id = "#dpCalendarModeDays";
+		const innerInput = await datepicker.getInnerInput();
+		await innerInput.click();
+		await browser.keys("2020, 04/01");
+		await browser.keys("Enter");
+
+		await datepicker.openPicker();
+		const calendar = await datepicker.getCalendar();
+
+		const currentPicker = await calendar.getProperty("_currentPicker");
+		assert.equal(currentPicker, "day", "calendar is opened on days");
+
+		const dayPicker = await calendar.shadow$("ui5-daypicker");
+		const monthPicker = await calendar.shadow$("ui5-monthpicker");
+		const yearPicker = await calendar.shadow$("ui5-yearpicker");
+		assert.notOk(await dayPicker.getAttribute("hidden"));
+		assert.ok(await monthPicker.getAttribute("hidden"));
+		assert.ok(await yearPicker.getAttribute("hidden"));
+
+		const timestamp_30_Jan_2020 = 1580342400;
+		const calendarDate_30_Jan_2020 = await datepicker.getPickerDate(timestamp_30_Jan_2020);
+		await calendarDate_30_Jan_2020.click();
+
+		assert.isFalse(await datepicker.isPickerOpen(), "picker is closed after day selection");
```

Comment:
what is the difference between notOk and isFalse? And should we mix them both in one test.

Reply:
there is huge difference between these, there is no problem in mixing them in one test when needed. 
https://api.qunitjs.com/assert/notOk/